Assets/Scripts/UI/UIHandler.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+[DefaultExecutionOrder(1000)]
```

Comment:
What! Didn't know about this attribute! Seems like it's [undocumented](https://forum.unity.com/threads/undocumented-but-public-defaultexecutionorder-attribute.530618/)... 🤯